app/ui-utils/client/lib/MessageAction.js

```diff
@@ -355,4 +356,24 @@ Meteor.startup(async function() {
 		order: 17,
 		group: 'menu',
 	});
+
+	MessageAction.addButton({
+		id: 'reaction-list',
+		icon: 'emoji',
+		label: 'Reaction_list',
+		context: ['message', 'message-mobile', 'threads'],
+		action() {
+			const { msg: { reactions } } = messageArgs(this);
+
+			modal.open({
+				template: createTemplateForComponent('reactionList', () => import('./reactionListContent')),
```

Comment:
```suggestion
				template: createTemplateForComponent('reactionList', () => import('./ReactionListContent')),
```

File name must match the component name case.